packages/frameworks/frameworks.json

```diff
@@ -4,70 +4,329 @@
   {
-    "name": "Svelte",
-    "slug": "svelte",
-    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/svelte.svg",
-    "tagline": "Svelte lets you write high performance reactive apps with significantly less boilerplate. ",
-    "website": "https://svelte.dev"
+    "name": "Docusaurus",
+    "slug": "docusaurus",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/docusaurus.svg",
+    "tagline": "Docusaurus makes it easy to maintain Open Source documentation websites.",
+    "detectors": {
+      "some": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"docusaurus\":\\s*\".+?\"[^}]*}"
+        },
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@docusaurus\\/core\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    }
+  },
+  {
+    "name": "Preact",
+    "slug": "preact",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/preact.svg",
+    "tagline": "Preact is a fast 3kB alternative to React with the same modern API.",
+    "website": "https://preactjs.com",
+    "detectors": {
+      "every": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"preact-cli\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    }
+  },
+  {
+    "name": "Ember",
+    "slug": "ember",
+    "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/ember.svg",
+    "tagline": "Ember.js helps webapp developers be more productive out of the box.",
+    "detectors": {
+      "every": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"ember-cli\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    }
   },
   {
     "name": "Vue.js",
     "slug": "vue",
     "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/vue.svg",
     "tagline": "Vue.js is a versatile JavaScript framework that is as approachable as it is performant.",
-    "website": "https://vuejs.org"
+    "website": "https://vuejs.org",
+    "detectors": {
+      "every": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@vue\\/cli-service\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    }
   },
   {
     "name": "Angular",
     "slug": "angular",
     "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/angular.svg",
     "tagline": "Angular is a TypeScript-based cross-platform framework from Google.",
-    "website": "https://angular.io"
+    "website": "https://angular.io",
+    "detectors": {
+      "every": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@angular\\/cli\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    }
   },
   {
-    "name": "Hugo",
```

Comment:
I thought Hugo was deleted but its just moved. How did you choose the order?

Reply:
I've sorted them after `frameworks.ts` in `@now/static-build`, to make sure they are detected in the same order